src/operations.rs

```diff
@@ -146,6 +146,31 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     }
 }
 
+#[inline(always)]
+pub(crate) fn add_in_length(enc: &mut u128, len: u64) {
+    #[cfg(all(target_feature = "sse2", not(miri)))]
```

Comment:
This refers to sse2 and below it refers to sss3, but it looks like these were both meant to be the same so as to provide two alternative paths.